### ISSUE TITLE
Add connection description to tooltip

### ISSUE THREAD

**Issue by danielhams** [danielhams/mad-java#7]
Would be nice to have a little description of what ports/wires a port is connected to when you mouse over it.

When a rack is large and you can't see the other end of the wire it's a pain to have to track it by eye.

Perhaps even have the possibility to (dis)connect directly from a menu as another way in parallel with dragging a wire.
